snippets/RPNSolver.md

```diff
@@ -0,0 +1,35 @@
+### RPNSolver
+
+Solves the given reverse polish notation
+```js
+const RPNSolver = RPN => {
+  const isNumeric = str => !isNaN(parseFloat(str)) && isFinite(str);
+  const isOperator = str => ['*','-','+','/','**'].includes(str)
+  let stack = [];
+  let solve  = RPN.replace(/\^/g,'**').split(/\s+/g);
+  solve.forEach(symbol => {
+    if(isNumeric(symbol)) {stack.push(symbol)}
+          else if (isOperator(symbol)){
+          a = stack.pop();
+          b = stack.pop();
```

Comment:
These two statements could be `[a,b] = [stack.pop(), stack.pop()]` in order to save a line (provided that works and I am not missing something).